common/ovf_export.yml

```diff
@@ -12,6 +21,13 @@
     export_with_images: "{{ ovf_export_with_iso | default(False) }}"
     export_dir: "{{ ovf_export_dst_path }}"
   register: ovf_export
+  until:
+    - ovf_export is defined
+    - ovf_export.failed is defined
+    - not ovf_export.failed
+  retries: "{{ ((ovf_export_timeout | int) / 5) | int }}"
```

Comment:
Here `ovf_export_timeout` is not the timeout of exporting template, it's more like retry times when exporting template failed. There is a parameter `download_timeout` in this module. Usually we do not add retry when task is "do config" like operation, if it fails, let it fail. Only add retry for "get config" like operation to avoid unclear issues.

Reply:
Thanks for pointing it out. Now it has been updated with download_timeout.